_posts/blog/2016-09-13-code-reviews.md

```diff
@@ -0,0 +1,91 @@
+
+
+## Don't Enforce Consensus
+
+Engineers often tend to forget that there is more than one valid opinion in a lot of cases. __Not everything is determined to false and true__.
+
+So if the reviewer has communicated his opinion appropriate and was not able to convince the reviewee, the code review is the wrong forum to let discussion escalate and enforce consensus. If the code is robust and fulfills the requirements, let it pass the review and try to move the topic out of the situation. You may have tech meetings or other forums in which you can discuss the topic (ideally in general and __not__ by specific example) with a broader audience.
+
+Letting the topic lie for a week can also help. Often times you realize that consensus __isn't important at all__. That not only there are more valid opinions than yours. But also that your team maybe can afford to __let both opinions stand__.
+
+## Don't Point At Rules - Give Explanations
+
+Rules and processes kill __creativity__. You want to give as much freedom to everyone as possible without hurting someone else (that's part of the German Constitution by the way ;)).
+
+Developers develop processes and methods. They align on conventions. And other developers violate those. Often times even conscientiously. How to handle those violations?
+
+Often times now is the time when the reviewer changes from consultant to finger-pointing judge (and hangman) who points at the wiki in which he documented the rule three years ago (without ever discussing it properly). Usually the tone gets rough. _Catch some breath, adjust your mindset..._.
+
+This behaviour is bad because creative people have the habit to question rules if they are in their way. If the result is that the rule does not create value (but supresses value-creation) in the case the engineer, striving to achieve a goal, will try to navigate around it. In such a case: __question the rule__, not the violator. In fact violating/questioning rules is a __healthy habit__.
+
+Feedback should be stated like this: _At some point we decided to approach things like this like that because we were convinced that's better because of reason X and Y. Don't you agree? Shall we re-evaluate?_
+
+Usually the topic is done now. The reviewee will probably accept the suggestion because the reviewer acted in a consulting role and took the reviewee serious, asked for his opinion.
+
+The reviewee will supposely not feel that the rule harms him so badly any more and follow it. Or a healthy discussion arises that in the end might change something for the better for the whole team.
+
+As a general rule of thumb: __do not define too many rules and processes__. And in a review: start with understanding the requirements, think about potential solutions also from a high-level/architectural/design-perspective. __How would you have solved the problem?__ Then look at the solution. __Find real bugs__. Only beginners focus on cosmetics (and usually miss real flaws).
```

Comment:
I think this is overuse of "/"
> high-level/architectural/design-perspective
 Suggestion:
> high-level, architectural and design perspective

You should consider making similar changes to some of the earlier instances where you've use "/" to combine concepts. Overuse can sometimes read like the author got a little lazy and could decide which concept they really meant; so you want to avoid it.